launchd/keybase.ios.release.plist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>keybase.ios.release</string>
+    <key>EnvironmentVariables</key>
+    <dict>
+        <key>GOPATH</key>
+        <string>/Users/test/go</string>
+        <key>SLACK_TOKEN</key>
+        <string></string>
+        <key>SLACK_CHANNEL</key>
+        <string></string>
+        <key>PATH</key>
+        <string>/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin</string>
+        <key>LOG_PATH</key>
+        <string>/Users/test/Library/Logs/ios.release.log</string>
+    </dict>
+    <key>ProgramArguments</key>
+    <array>
+        <string>/bin/bash</string>
+        <string>/Users/test/go/src/github.com/keybase/client/packaging/ios/build_and_publish.sh</string>
```

Comment:
instead of calling sh from client directly .. i think you want to make a copy/version of build.sh in launchd dir here... e.g. if there is an error it reports it... also success too